staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -2585,10 +2590,19 @@ type NodeMediatedDeviceTypesConfig struct {
 	// +listType=atomic
 	MediatedDevicesTypes []string `json:"mediatedDevicesTypes,omitempty"`
 	// +optional
-	// +listType=atomic
+	//+listType=atomic
```

Comment:
Probably better to not do that, all other comments do start with a space

Reply:
Mmmmmh, this could be an outcome of a rebase. Thanks!